API.md

```diff
@@ -0,0 +1,53 @@
+# GPIO Expander Pi API
+
+## `class GpioExpander`
+
+Create an object `GpioExpander` to communicate with a particular GpioExpander boards.
+
+### `GpioExpander(i2c_address=42)`
+
+The constructor for GpioExpander class. The `i2c_address` specifies the board
+I²C address which is `42` factory-default but can be changed programmatically.
+
+### `pin_mode(pin: int, mode: int) -> None`
+
+Configures the operation `mode` of the `pin`:
+
+- INPUT
+- INPUT_PULLUP
+- INPUT_PULLDOWN
+- OUTPUT
```

Comment:
What’s this? True enum, semi-enum, globals? Where are they defined? Use fully qualified names, please.

Reply:
I think we can add these constants as static members directly to the GpioExpander class. Then we can call the `pin_mode` method as:
`obj.pin_mode(pin, GpioExpander.INPUT)'